### DISCUSSION TITLE
Can't start the service after updating to 2.1.0

### DISCUSSION THREAD

**Discussion by onmyouji** [DNSCrypt/dnscrypt-proxy#1825] [Q&A] [answered]
Hi all,

I can't start the service after updating to 2.1.0

Got this error message 
Error 1067: The process terminated unexpectedly

Tried to uninstall the service and restarting the system, but it's still not working.

I'm on Windows 10 Home 64-bit 20H2 19042.1151

Anyone else have the same issue?

**Comment by ianbashford** [comment, marked as answer]
I've just tried a fresh install and it started up OK
Download and unzip -- then in powershell

``` .\dnscrypt-proxy.exe -config .\example-dnscrypt-proxy.toml  ```

I'd try that, and see if it works.   You could also try the same with your current install, and see what output you get on the screen - you should get some good feedback from the logging.  
 [a common issue can be the old process not stopping, so it fails to bind to port 53 on startup as there's already a process there].

**Reply by onmyouji** [reply]
Thank you.

It works now after using fresh new .toml

Seems like there's something in my current .toml config file that prevents the service from running.

Thank you for your help.
